fuzzers/libafl-fuzz/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "libafl-fuzz"
+version = "0.0.1"
+edition = "2021"
+
+[dependencies]
+clap = { version = "4.5.8", features = ["derive", "env"] }
```

Comment:
I'd try to make sure this is the same version as we use everywhere else in the repo, else we may pull in the decpendency twice

Reply:
ah sorry i stashed these changes and forgot to unstash them before pushing